frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -37,117 +34,89 @@ end
 function BookInfo:show(file, book_props)
     local kv_pairs = {}
 
+    -- File section
     local directory, filename = util.splitFilePathName(file)
-    local filename_without_suffix, filetype = util.splitFileNameSuffix(filename) -- luacheck: no unused
-    if filetype:lower() == "zip" then
-        local filename_without_sub_suffix, sub_filetype = util.splitFileNameSuffix(filename_without_suffix) -- luacheck: no unused
-        sub_filetype = sub_filetype:lower()
-        local supported_sub_filetypes = { "fb2", "htm", "html", "log", "md", "txt" }
-
-        for __, t in ipairs(supported_sub_filetypes) do
-            if sub_filetype == t then
-                filetype = sub_filetype .. "." .. filetype
-                break
-            end
-        end
-    end
-    local file_size = lfs.attributes(file, "size") or 0
-    local file_modification = lfs.attributes(file, "modification") or 0
+    local __, filetype = filemanagerutil.splitFileNameType(filename)
+    local attr = lfs.attributes(file)
+    local file_size = attr.size or 0
     local size_f = util.getFriendlySize(file_size)
     local size_b = util.getFormattedSize(file_size)
-    local size = string.format("%s (%s bytes)", size_f, size_b)
     table.insert(kv_pairs, { _("Filename:"), BD.filename(filename) })
     table.insert(kv_pairs, { _("Format:"), filetype:upper() })
-    table.insert(kv_pairs, { _("Size:"), size })
-    table.insert(kv_pairs, { _("File date:"), os.date("%Y-%m-%d %H:%M:%S", file_modification) })
+    table.insert(kv_pairs, { _("Size:"), string.format("%s (%s bytes)", size_f, size_b) })
+    table.insert(kv_pairs, { _("Last read date:"), os.date("%Y-%m-%d %H:%M:%S", attr.access) })
+    table.insert(kv_pairs, { _("Date modified:"), os.date("%Y-%m-%d %H:%M:%S", attr.modification) })
```

Comment:
The wording is a bit inconsistent (`Last read date:` vs `Date modified:`, shouldn't it be `Modified date:` ?
But I would keep using `File date:` (modification time is the Linux/Lua technical name for the info ; it's the creation date or the copy date for copied files depending on the tech used to do that copy, or the modification time for PDF with highlights written into them).

But: `attr.access` may or may not be updated by the OS when the file is read from. And "read from" may be just when extract and cache book info, or fetching book info, and it's not really the "last read date" as we expect it in a book reading application :)
For a field name "Last read date", I would expect the date to come from the doc settings file data.

I think the only valid and interesting info is attr.modification, and it should just be named "File date" to not imply anything else.
(Even if my Windows 10 Explorer shows "Modifié le" ("Modified at" in English I guess) :)

Reply:
I think Windows created (birth) and modified are much more useful than access and change time. To me those others aren't useful.

Reply:
`lfs.attributes` doesn't provide "birth" date. So, show `modification` only?

Reply:
`atime` will very much be useless 99.99% of the time, so, yeah, +1 for only mentioning `mtime` ;).

Reply:
And to sate my curiosity about `btime`, it's a Linux extension, only available when calling `statx`, which requires Linux 4.11+ & glibc 2.28+ ;).

(It's implemented in stat on BSDs, and apparently ntfs3g, too).

Reply:
btime usually seems to be the same as mtime anyway due to modern programs taking the safe approach (though on Win created usually seems to be maintained)